From f747c5b5f35490ff177b8327ff1f7299f7d0bf4a Mon Sep 17 00:00:00 2001
From: Naushir Patuck <naush@raspberrypi.com>
Date: Mon, 11 May 2020 13:06:27 +0100
Subject: [PATCH] dt-bindings: bcm2835-unicam: Update documentation
 with new clock params

Update the documentation to reflect the new "VPU" clock needed
by the bcm2835-unicam driver.

Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
---
 .../devicetree/bindings/media/bcm2835-unicam.txt          | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/Documentation/devicetree/bindings/media/bcm2835-unicam.txt
+++ b/Documentation/devicetree/bindings/media/bcm2835-unicam.txt
@@ -20,7 +20,7 @@ Required properties:
 - interrupts	: should contain the IRQ line for this Unicam instance.
 - clocks	: list of clock specifiers, corresponding to entries in
 		  clock-names property.
-- clock-names	: must contain an "lp" entry, matching entries in the
+- clock-names	: must contain "lp" and "vpu" entries, matching entries in the
 		  clocks property.
 
 Unicam supports a single port node. It should contain one 'port' child node
@@ -46,9 +46,9 @@ Example:
 		reg = <0x7e801000 0x800>,
 		      <0x7e802004 0x4>;
 		interrupts = <2 7>;
-		clocks = <&clocks BCM2835_CLOCK_CAM1>;
-		clock-names = "lp";
-
+		clocks = <&clocks BCM2835_CLOCK_CAM1>,
+			 <&firmware_clocks 4>;
+		clock-names = "lp", "vpu";
 		port {
 			csi1_ep: endpoint {
 				remote-endpoint = <&tc358743_0>;
